[iOS] Fix final state of canceled animations (#2532)
unoplatform/uno: doc/articles/uno-development/working-with-the-samples-apps.md

The Uno solution provides a set of sample applications that provide a way to test features, as
well as provide a way to write UI Tests.

> Those applications are structured in a way that samples can created out of normal `UserControl` instances, 

Reviewer: ```suggestion
Those applications are structured in a way that samples can be created out of normal `UserControl` instances, 
```

> marked with the `SampleAttribute` (and previous `SampleControlInfoAttribute`) so the sample application can discover them.

Reviewer 2: https://sembr.org/

Author: It's compliant to point 6:
> A semantic line break MAY occur after a dependent clause in order to clarify grammatical structure or satisfy line length constraints.

To create a UI Test for the sample applications:
- Create or reuse a folder named from the namespace of the control or class your want to test, replacing "`.`" by "`_`"
- Create a new `UserControl` from the Visual Studio templates in the `UITests.Shared` project
- Add `[Uno.UI.Samples.Controls.Sample]` on the code-behind class. 

>   _Note: This attribute also accepts few other metadata about you sample like a list of `Categories`, a `Name`,

Reviewer: ```suggestion
  _Note: This attribute also accepts a few other metadata about your sample, such as a list of `Categories`, a `Name`,
```
